apps/web/src/ee/clerk/providers/ClerkProvider.tsx

```diff
@@ -17,13 +17,27 @@ const ClerkModalElement = {
   rootBox: {
     width: 'auto',
   },
+  navbarButton: {
+    fontFamily: 'var(--nv-fonts-system)',
+    fontWeight: 'var(--nv-font-weights-strong)',
+    fontSize: 'var(--nv-font-sizes-88)',
+    margin: '10px 10px 10px 10px important!',
```

Comment:
```suggestion
    margin: 'var(--nv-spacing-75) !important',
```
This will result in 12px (0.75rem). 10px is not valid in our design system

Reply:
applied